static/base.css

```diff
@@ -1801,9 +1801,19 @@ ul.nav.navbat-nav .btn-outline-secondary:hover{
     visibility: hidden;
     padding-left: 3px;
     margin-bottom: -0.5px;
-    pointer-events: none;
+    overflow-x: auto;
+    overflow-y: hidden;
+}
+
+#statusBar #scrollContainer{
+    width: max-content;
 }
 
+#statusBar::-webkit-scrollbar{
```

Comment:
It seems like webkit-scrollbar doesn't work on Firefox.

https://caniuse.com/?search=webkit-scrollbar

But it doesn't cause an issue, just gets ignored, and I can't find a mozilla equivalent. So I think it's OK.

Reply:
huh interesting. looks like the firefox scrollbar for that element is perfect though. so luckily it doesnt need that css statement at all